service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,7 +4,21 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-If you selected the automatic update stream, updating {ProductName} doesn't require any extra steps. 
+If you're updating from {ProductName} 1.0 to 1.1, you must update the `ServiceMeshControlPlane` resource YAML file to enable the update. 
+
+. In the web console, click the {ProductName} operator.
+
+. Click the *Project* menu and choose the project where your `ServiceMeshControlPlane` is deployed from the list, for example `istio-system`.
+
+. Click the name of your control plane, for example `basic install`.
+
+. Click YAML and add `version 1.1` to the spec of your `ServiceMeshControlPlane` resource YAML file. 
+
+    spec:
+      version: v1.1
```

Comment:
Should we add some general docs about the `version` field?  It can be set to either `v1.1` or `v1.0` and defaults to the latest version available, currently `v1.1`.  `v1.0` corresponds to Maistra 1.0, which is our distribution of Istio `1.1.x`, and `v1.1` corresponds to Maistra 1.1, which is our distribution of Istio `1.4.x`.

Reply:
+1